source/MRMesh/MRContoursCut.h

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include "MRVector3.h"
+#include "MRAffineXf3.h"
 #include "MRId.h"
 #include "MRBitSet.h"
```

Comment:
Please consider minimizing the number of incudes in the header. Forward declarations shall be enough

Reply:
Then we couldn't make a default parameter for the transformation (`const AffineXf3f& xf = {}`)

Reply:
Ok, and what about other includes here?

Reply:
ok, let's do the cleaning in a separate PR